idd/Energy+.idd.in

```diff
@@ -56742,6 +56742,7 @@ Coil:WaterHeating:Desuperheater,
   A7 , \field Tank Object Type
        \type choice
        \key WaterHeater:Mixed
+       \key WaterHeater:Stratified
        \default WaterHeater:Mixed
        \note Specify the type of water heater tank used by this desuperheater water heating coil.
   A8 , \field Tank Name
```

Comment:
Don't you also need to include `\reference DesuperHeatingCoilSources` in the Name field of the `Coil:Cooling:WaterToAirHeatPump:EquationFit` object?